tools/archetype/binding.test/src/main/resources/archetype-resources/__artifactId__.launch

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
there used to be a list of "selected target plugins" - they were important, as right now the OSGi tests fail in the IDE. The symptom is that the bundleContext is `null`, which is because the bundle under test is not started, and tons of others also aren't.